src/base/base.ts

```diff
@@ -79,8 +79,13 @@ export default class Base {
    * @returns {Promise<GetResponse>}
    */
   public async get(key: string): Promise<GetResponse> {
-    const trimmedKey = key.trim();
-    if (!trimmedKey.length) {
+    let trimmedKey;
+    if (key) {
+      trimmedKey = key.trim();
+      if (!trimmedKey.length) {
+        throw new Error('Key is empty');
+      }
+    } else {
```

Comment:
Hey instead of doing this can you please do this
```
const trimmedKey = key?.trim();
if (!trimmedKey) {
  throw new Error('Key is empty');
}
```

Reply:
Fixed